iphone/Maps/Maps.xcodeproj/project.pbxproj

```diff
@@ -4418,6 +4419,27 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		ED7077692DDDE6120028271D /* Run SwiftLint */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
```

Comment:
Is sandboxing needed at all btw? Will adding ${SRCROOT}/../.swiftlint.yml as input path help?